### DISCUSSION TITLE
Change Brightness via CLI on webOS

### DISCUSSION THREAD

**Discussion by Cptn-Reynolds** [awawa-dev/HyperHDR#554] [Q&A] [answered]
Good day,

I'm running HyperHDR on webOS and would like to map different brightness levels (the value in "LED Hardware" -> "Brightness" in the webinterface) to different buttons on my LG remote. I found a homebrew tool which can map custom code to a button, but can't seem to figure out which files to manipulate / overwrite, where the settings on HyperHDR are stored.

Can you give me a nudge in the right direction? Highly appreciated :)

**Comment by awawa-dev** [comment, marked as answer]
Hi
There is currently no access to the led property via the api, but you can use the JSON API to modify the LED brightness in the processing ("brightness" or "Luminance" depending on whether you have classic calibration on/off). The wizard will build a command link or POST packet for you:
![obraz](https://user-images.githubusercontent.com/69086569/233446897-78547e67-cf1a-41a2-aa1a-52096926de7d.png)

**Comment by Cptn-Reynolds** [comment]
Genius, thanks for that info :) Somehow must have overlooked the JSON section

**Comment by AbbieDoobie** [comment]
Is there a way to _increment_ (or decrement) by X amount? So specifically, instead of setting LED brightness to X, increase/decrease it by X.

I would like to bind keys to increase and decrease brightness.

**Reply by awawa-dev** [reply]
There is currently no such functionality directly, and to achieve such an effect you can read the current brightness and set it decreased/increased by X or set some value at the start and remember it, and then send it appropriately changed and remember it as well.
